src/main/java/com/marksem/crm/entity/Booking.java

```diff
@@ -0,0 +1,31 @@
+package com.marksem.crm.entity;
+
+import lombok.*;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@EqualsAndHashCode(callSuper = true)
+@Entity
+@Table(name = "booking")
+@Data
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+@EntityListeners(AuditingEntityListener.class)
+public class Booking extends BaseEntity implements Serializable {
+    @Basic
+    @Temporal(TemporalType.TIMESTAMP)
+    private java.util.Date fromDate;
```

Comment:
import `java.util.Date` class everywhere so you don't have to write it down for each field